vllm/engine/output_processor/stop_checker.py

```diff
@@ -100,20 +104,21 @@ def _check_stop_strings(seq: Sequence, new_char_count: int,
         for stop_str in sampling_params.stop:
             stop_string_len = len(stop_str)
             # Avoid searching already-searched text.
-            stop_index = seq.output_text.find(
-                stop_str, -new_char_count - stop_string_len)
+            stop_index = output_text.find(stop_str,
+                                          -new_char_count - stop_string_len)
             if stop_index == -1:
                 continue
 
             if sampling_params.include_stop_str_in_output:
                 # Truncate to end of stop string.
                 stop_index += stop_string_len
-                if stop_index >= len(seq.output_text):
+                if stop_index >= len(output_text):
                     # No truncation required.
-                    return stop_str
+                    return stop_str, -1
 
             # Truncate the output text to either the beginning
             # or end of the stop string.
-            seq.output_text = seq.output_text[:stop_index]
-            return stop_str
+            return stop_str, stop_index
         return None
+
+        #seq.output_text = seq.output_text[:stop_index]
```

Comment:
Oops, I will hold off removing this so as to not trigger another round of CI tests (until needed for other changes)